src/light-controller.js

```diff
@@ -53,6 +53,8 @@ export class LightController extends Controller {
   }
   get _max() {
     switch (this.attribute) {
+      case "brightness":
+        return this._config.max || 100
```

Comment:
This is not necessary. `this.max` will only get `this._max` if `this._config.max` is not set. See controller.js.

Reply:
Right, I changed it